src/Microsoft.Azure.SignalR.Common/Exceptions/AzureSignalRInaccessibleEndpointException.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.Azure.SignalR.Common
+{
+    [Serializable]
+    public class AzureSignalRInaccessibleEndpointException : AzureSignalRException
+    {
+        private const string ErrorMessage = "Unable to access SignalR service. May caused by one or more of the following reasons: Incorrect endpoint or DNS error.";
```

Comment:
I'm still thinking the possible reason may not be accurate. The inner exception may be self-explain enough.

Reply:
The inner exception exception is `HttpRequestException`, the message is "No such host is known" (in Windows) and "No such device or address " (in Linux). I think the inner exception is not enough and message is different in different platform.

Reply:
@chenkennt How about this?

message format: phenomenon + reason in inner exception + recommend action

$"Unable to access SignalR service. {innerExceptionMessage}. Please make sure the endpoint or DNS setting is correct."